lib/active_stash.rb

```diff
@@ -14,4 +14,27 @@
 
 module ActiveStash
   class Error < StandardError; end
+
+  class Config
+    def to_client_opts
+      self.instance_values
+        .map { |key, value| [key.to_s.camelize(:lower).to_sym, value] }
+        .to_h
+        .compact
+    end
+  end
+
+  Config.class_eval do
+    CipherStash::Client.client_options.each do |option|
+      attr_accessor option.to_s.underscore.to_sym
```

Comment:
Not sure if this is too clever, but it allows for errors during app init if the user provides an invalid config key.

We could check the options when the StashRB client is constructed, but I figured it'd be better to fail as early as possible.

Reply:
It's pretty clever.  Since `attr_accessor` can take many arguments, you can make this into a one-liner with `attr_accessor *CipherStash::Client.client_options.map { |o| o.to_s.underscore.to_sym }` (or something fairly similar thereto).  If you don't mind multiple iteration, you can get rid of the braces with `.map(&:to_s).map(&:underscore).map(&:to_sym)`, although I don't personally think that's more readable.

Reply:
Nice, updated